minimesos/src/main/java/com/containersol/minimesos/mesos/Registrator.java

```diff
@@ -0,0 +1,50 @@
+package com.containersol.minimesos.mesos;
+
+import com.containersol.minimesos.cluster.MesosCluster;
+import com.containersol.minimesos.config.RegistratorConfig;
+import com.containersol.minimesos.container.AbstractContainer;
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerCmd;
+import com.github.dockerjava.api.model.Bind;
+
+public class Registrator extends AbstractContainer {
```

Comment:
I'm not against comments (usually), but commenting the obvious does not make sense to me. Check out the other AbstractContainer classes. 
```
/**
 * Base zookeeper class
 */
public class ZooKeeper extends AbstractContainer {
```

Does this comment really add something to our understanding of what this class does? The rest is the same. 

What do you think?